index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
```

Comment:
- [ ] Kindly check on your project design after implementing changes requested the design got ruined , below is a comparison
`Before`
![image](https://user-images.githubusercontent.com/61408860/185430657-39d83274-0515-4615-9cb7-3d4a028f8d4a.png)
`After`
![image](https://user-images.githubusercontent.com/61408860/185430744-0f8089e3-e02b-47b0-9ed3-0cb0bd80ad34.png)